go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/openshift/cert-manager-operator-release
 
-go 1.23.0
+go 1.23.9
 
 require github.com/mikefarah/yq/v4 v4.45.1
```

Comment:
nit: Please update this to `github.com/mikefarah/yq/v4@v4.45.2` as well.

Reply:
Done, updated.